docs/pages/technical-specs/expo-updates-1.mdx

```diff
@@ -60,47 +68,81 @@ A conformant client library configured to perform [code signing](#code-signing)
+- `expo-server-defined-headers` is an [Expo SFV](expo-sfv-0.mdx) dictionary. It defines headers that a client library MUST store until overwritten by a newer dictionary, and they MUST be included in every subsequent [update request](#request).
 - `cache-control` MUST be set to an appropriately short period of time. A value of `cache-control: private, max-age=0` is recommended to ensure the newest manifest is returned. Setting longer cache ages could result in stale updates.
-- `content-type` MUST be determined by _proactive negotiation_ as defined in [RFC 7231](https://tools.ietf.org/html/rfc7231#section-3.4.1). Since the client library is [required](#manifest-request) to send an `accept` header with each manifest request, this will always be either `application/expo+json`, `application/json`; otherwise the request would return a `406` error.
+- `content-type` MUST be determined by _proactive negotiation_ as defined in [RFC 7231](https://tools.ietf.org/html/rfc7231#section-3.4.1). Since the client library is [required](#request) to send an `accept` header with each manifest request, this will always be either `application/expo+json`, `application/json`; otherwise the request would return a `406` error.
+
+### Other Response Headers
+
+```
+expo-signature: *
+```
+
 - `expo-signature` SHOULD contain the signature of the manifest to be used during the validation step of [code signing](#code-signing) if the request for the manifest contained the `expo-expect-signature` header. This is an [Expo SFV](expo-sfv-0.mdx) dictionary which MAY contain any of the following key value pairs:
   - `sig` MUST contain the signature of the manifest. The name of this field matches that of `expo-expect-signature`.
   - `keyid` MAY contain the keyId of the key the server used to sign the response. The client SHOULD use the certificate that matches this `keyid` to verify the signature.
   - `alg` MAY contain the algorithm the server used to sign the response. The client SHOULD use this field only if it matches the algorithm defined for the certificate matching `keyid`.
 
-### Manifest Response Body
+### Multipart Response
+
+A update response of this format is defined by the `multipart/mixed` MIME type as defined by [RFC 2046](https://tools.ietf.org/html/rfc2046#section-5.1).
+
+Headers for this response format are the [common response headers](#common-response-headers), with the following exceptions:
+
+- `content-type` SHOULD have a `multipart/mixed` value as defined by [RFC 2046](https://tools.ietf.org/html/rfc2046#section-5.1)
+
+Part order is not strict. A multipart response with no parts should be considered a no-op (no updates or directives available), though headers SHOULD be sent nevertheless.
```

Comment:
Let's get this PR through but one thing we need to resolve in a follow-up is whether server-defined headers should be set when the server responds with an empty multipart response.If the client is expected to persist the server-defined headers, then no-op isn't the right word. 

We should take a look at expo-updates to see what it does today as well as our intent/motivation behind expo-server-defined-headers and then tidy up the spec. It's already kind of ambiguous about how to treat server-defined headers outside of the happy path where the server responds with a manifest.